docs/install.rst

```diff
@@ -0,0 +1,31 @@
+.. _topics-install:
+
+=======
+Install
+=======
+
+.. note:: As a general rule, it's recommended to use an isolation tool such as ``virtualenv``.
```

Comment:
I would recommend including the virtualenv steps in the install instructions. It's pretty much universally a good idea, so we want to encourage folks to do it by default. This note is still helpful (gives the opt-out if they don't want to use it), but for anyone unfamiliar with python, we should encourage them to work in a properly isolated env. Those familiar enough to make the choice themselves will be able to cut out the steps they don't need.